.pre-commit-config.yaml

```diff
@@ -10,3 +10,4 @@ repos:
   - id: double-quote-string-fixer
   - id: end-of-file-fixer
   - id: flake8
+    exclude: .git,docs,env,venv,.ropeproject,build,.tox
```

Comment:
pre-commit will only ever run on checked in files, and takes a [regular expression](https://pre-commit.com/#regular-expressions) you want `exclude: ^docs/`